webcompat/templates/list-issue/issuelist-search.jst

```diff
@@ -1,20 +1,10 @@
 <script type="text/template">
-<!--
-      <label for="js-SearchForm-input" class="is-hidden">Search By Keyword</label>
-      <input type="text" id="js-SearchForm-input" class="form-field text-field is-hidden mousetrap" placeholder="Search by keyword">
-      <button role="button" class="nav-button nav-link js-SearchForm-button" title="Search for an item.">
-        <svg class="icon nav-icon" viewBox="0 0 30 30" role="presentation">
-            <use href="#svg-search2">Search</use>
-        </svg>
-    </button>
---->
   <form id="x-search-bar" method="get" action="/issues">
         <input class="form-field text-field" name="q" placeholder="Search issues by keyword"/>
-        <button class="nav-button js-SearchBarClose" aria-hidden="true" type="button">
+        <button role="button" class="nav-button nav-link js-SearchForm-button" title="Search for an item.">
```

Comment:
Why are we removing `type=button`? Does that change behavior to do a form submit?

Reply:
this definitely looks like a mistake. will fix.